### ISSUE TITLE
[native-image] java.lang.UnsatisfiedLinkError: jdk.internal.misc.VM.getNanoTimeAdjustment on Windows

### ISSUE THREAD

**Issue by znbang** [oracle/graal#1871]
Tested with graalvm-ce-java11-windows-amd64-19.3.0.

Test.java:
```java
public class Test {
  public static void main(String[] args) throws Exception {
    java.time.LocalDate.now();
  }
}
```

Error messages:
```
C:\>java -version
openjdk version "11.0.5" 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-jvmci-19.3-b05-LTS)
OpenJDK 64-Bit GraalVM CE 19.3.0 (build 11.0.5+10-jvmci-19.3-b05-LTS, mixed mode, sharing)

C:\>javac Test.java

C:\>native-image Test Test
[Test:13060]    classlist:   3,746.77 ms
[Test:13060]        (cap):   6,499.36 ms
[Test:13060]        setup:   9,854.62 ms
[Test:13060]   (typeflow):  11,461.98 ms
[Test:13060]    (objects):   6,540.73 ms
[Test:13060]   (features):     557.49 ms
[Test:13060]     analysis:  19,074.34 ms
[Test:13060]     (clinit):     382.82 ms
[Test:13060]     universe:   1,337.33 ms
[Test:13060]      (parse):   2,904.98 ms
[Test:13060]     (inline):   2,764.40 ms
[Test:13060]    (compile):  14,443.57 ms
[Test:13060]      compile:  20,949.44 ms
[Test:13060]        image:   1,862.46 ms
[Test:13060]        write:     916.49 ms
[Test:13060]      [total]:  58,264.48 ms

C:\>Test
Exception in thread "main" java.lang.UnsatisfiedLinkError: jdk.internal.misc.VM.getNanoTimeAdjustment(J)J [symbol: Java_jdk_internal_misc_VM_getNanoTimeAdjustment or Java_jdk_internal_misc_VM_getNanoTimeAdjustment__J]
        at com.oracle.svm.jni.access.JNINativeLinkage.getOrFindEntryPoint(JNINativeLinkage.java:145)
        at com.oracle.svm.jni.JNIGeneratedMethodSupport.nativeCallAddress(JNIGeneratedMethodSupport.java:57)
        at jdk.internal.misc.VM.getNanoTimeAdjustment(VM.java)
        at java.time.Clock$SystemClock.instant(Clock.java:529)
        at java.time.LocalDate.now(LocalDate.java:229)
        at java.time.LocalDate.now(LocalDate.java:198)
        at Test.main(Test.java:3)
```

**Comment by legendaryhero1981** [comment]
My machine is also the same error, look forward to the project team as soon as possible!

**Comment by olpaw** [comment]
@legendaryhero1981 @znbang can you still reproduce the issue when you use the latest 20.0 snapshot build: https://github.com/graalvm/graalvm-ce-dev-builds/releases/tag/20.0.0-dev-20200213_2110

**Comment by znbang** [comment]
@olpaw it's solved in 20.0 snapshot, thanks
